<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamLadderDialog</class>
 <widget class="QDialog" name="TeamLadderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="3">
    <widget class="QTextEdit" name="TextBox">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="TextLine">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2v2 Ladder Team Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="labelTeammate">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pick a faction. Click Ready when your done. A suitable team will be found for you.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item alignment="Qt::AlignVCenter">
      <widget class="QLabel" name="FactionGraphic1">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QComboBox" name="FactionSelect">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>Random</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UEF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cybran</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aeon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seraphim</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="FactionFriend">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Random</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QLabel" name="FactionGraphic2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="StatusLabel">
     <property name="text">
      <string>Finding opponent team...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="TeamInfo">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>teaminfo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item alignment="Qt::AlignVCenter">
      <widget class="QCheckBox" name="CheckReady">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ready</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>135</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>65</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>           Ready</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QCheckBox" name="FriendReady">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
